libs/etterlatte-ktor/build.gradle.kts

```diff
@@ -28,8 +28,11 @@ dependencies {
 
     implementation(libs.navfelles.tokenvalidationktor2)
 
+    implementation(libs.metrics.micrometer.prometheus)
     implementation(libs.metrics.prometheus.simpleclientcommon)
     implementation(libs.metrics.prometheus.simpleclienthotspot)
+    implementation("io.ktor:ktor-server-core-jvm:2.3.1")
+    implementation("io.ktor:ktor-server-metrics-micrometer-jvm:2.3.1")
```

Comment:
Burde rydde opp i avhengigheter. Ganske sikker på at du bare trenger disse:

```
    implementation(libs.metrics.micrometer.prometheus)
    implementation("io.ktor:ktor-server-metrics-micrometer-jvm:2.3.1")
``` 

Den for Ktor finnes for øvrig i versjonsbiblioteket vårt under navnet `ktor2-metricsmicrometer`

Reply:
Jepp, noe default greier i i intellij som har skjedd her :D

Reply:
208d32dea015bb93048366965ff11afb524389b7